entries/jQuery.grep.xml

```diff
@@ -4,7 +4,7 @@
   <desc>Finds the elements of an array which satisfy a filter function. The original array is not affected.</desc>
   <signature>
     <added>1.0</added>
-    <argument name="array" type="Array">
+    <argument name="array" type="ArrayLikeObject">
       <desc>The array to search through.</desc>
```

Comment:
@yadimon Can you improve this sentence by specifying that it can be also an Array-like object?